src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ViewPenaltiesController.java

```diff
@@ -84,15 +87,17 @@ public String getViewPenalties(@PathVariable String companyNumber,
             return ERROR_VIEW;
         }
 
-        model.addAttribute("outstanding", lateFilingPenalty.getOutstanding());
-        model.addAttribute("madeUpDate",
-                LocalDate.parse(lateFilingPenalty.getMadeUpDate(),
-                        DateTimeFormatter.ofPattern("uuuu-MM-dd", Locale.UK))
-                        .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK)));
+        DecimalFormat formatter = new DecimalFormat("#,###");
```

Comment:
Can you make this a constant variable please, `private static final`?
Also we can just move the `.format` operation to the `model.addAttribute` line - no need for String variable?

Reply:
Going to move this stuff to penaltyUtils class as well and I'll make it `private static final ` in there